test/specs/scenarios/accessibility.js

```diff
@@ -0,0 +1,184 @@
+import { describe, it } from '../../utils/mochaw'
+import { localhostUrl } from '../../config.json'
+import { goToPassportUploadScreen, uploadFileAndClickConfirmButton } from './sharedFlows.js'
+import { runAccessibilityTest } from '../../utils/accessibility'
+
+const options = {
+  pageObjects: [
+    'Welcome',
+    'Confirm',
+    'DocumentSelector',
+    'DocumentUpload',
+    'CrossDeviceIntro',
+    'CrossDeviceLink',
+    'CrossDeviceMobileConnected',
+    'CrossDeviceSubmit',
+    'LivenessIntro',
+    'PoaDocumentSelection',
+    'PoaGuidance',
+    'PoaIntro',
+    'VerificationComplete',
+    'BasePage'
+  ]
+}
+
+export const accessibilityScenarios = async(lang='en') => {
+
+  describe(`ACCESSIBILITY scenarios in ${lang}`, options, ({driver, pageObjects}) => {
+
+    const {
+      welcome,
+      confirm,
+      documentSelector,
+      documentUpload,
+      crossDeviceIntro,
+      livenessIntro,
+      poaDocumentSelection,
+      poaIntro,
+      basePage
+    } = pageObjects
+
+    const copy = basePage.copy(lang)
+
+    const goToPoADocumentSelectionScreen = async () => {
+      driver.get(localhostUrl + `?poa=true&async=false&useWebcam=false`)
+      welcome.primaryBtn.click()
+      poaIntro.clickStartVerificationButton()
+    }
+
+    const goToCrossDeviceScreen = async () => {
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      crossDeviceIntro.continueButton.click()
+    }
+    
+    //Welcome
+    it('should verify accessibility for the welcome screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify focus management for the welcome screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.verifyFocusManagement()
+    })
+
+    //Cross Device Sync
+    it('should verify accessibility for the cross device intro screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
```

Comment:
Minor: this is pretty much the `goToCrossDeviceScreen()` steps aside from the last "Click Continue button" in that function